### PR TITLE
Fix failing test with interactive menus

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix failing test with interactive menus.

Note: Since the mattermost instance both in local and test server interacts with each other via local network, it's no longer necessary to use full domain name as `webhookBaseUrl ` as it will only complicate things. 

Carlos fixed the webhook server in CI.

#### Ticket Link
n/a

### CODE REVIEW

File: e2e/cypress.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "baseUrl": "http://localhost:8065",
+    "webhookBaseUrl": "http://localhost:3000",
```
Comment (thekiiingbob) [inline e2e/cypress.json]: I think we should only put valid Cypress settings in this file so it's clear that these options are for configuring cypress. We can use the `env` section of this file for variables like this, IMO.

Comment (saturninoabril) [reply e2e/cypress.json]: I agree, see updated commit.  Thanks

**Comment by saturninoabril** [comment]
Do you have webhook server running?  It should be started manually (`npm run start:webhook`) if you're testing it against baseUrl of `localhost:8065`.

**Comment by prapti** [comment]
> Do you have webhook server running? It should be started manually (`npm run start:webhook`) if you're testing it against baseUrl of `localhost:8065`.

That was it!

**Comment by prapti** [comment]
(Sorry I mistakenly hit close and comment)

**Pull request merged**